packages/modeling/src/operations/extrusions/extrudeHelical.js

```diff
@@ -0,0 +1,81 @@
+const { TAU } = require('../../maths/constants')
+const slice = require('./slice')
+const mat4 = require('../../maths/mat4')
+const extrudeFromSlices = require('./extrudeFromSlices')
+const geom2 = require('../../geometries/geom2')
+
+/**
+ * Perform a helical extrude of the geometry, using the given options.
+ *
+ * @param {Object} options - options for extrusion
+ * @param {Number} [options.angle=TAU] - angle of the extrusion (RADIANS) positive for right-hand rotation, negative for left-hand
+ * @param {Number} [options.pitch=10] - elevation gain for each turn
```

Comment:
I like both 'pitch' and 'height'. I can see some users wanting to specify the pitch for each turn, but some users may want to set a final 'height'. I'm wondering if we could support both of these.